### DISCUSSION TITLE
how can i change session object from storefront?

### DISCUSSION THREAD

**Discussion by Nurtilekk** [nextauthjs/next-auth#2832] [Help]
Im taking user`s avatar from session object. Also i have uploading avatar logic on profile page. The problem is when user is uploading new img, session object doesnt change, thus i cannot show new uploaded photo of customer. How can i forcefully change session object, any example?

**Comment by balazsorban44** [comment]
no easy way for jwt cookie persisted sessions, without triggering a re-login. follow #2269 for more info on this.

**Reply by Nurtilekk** [reply]
maybe i can try something like this 
![image](https://user-images.githubusercontent.com/62701301/134760629-ba91ed29-6886-47f5-a9e4-38b3f9e2c850.png)
but it sends error 
![image](https://user-images.githubusercontent.com/62701301/134760650-86486ffd-9515-4b1d-a6ca-0e5eb1c392d3.png)

**Reply by balazsorban44** [reply]
hard to say what the problem is without seeing your code
